nifi-extension-bundles/nifi-aws-bundle/nifi-aws-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -103,6 +103,11 @@ The following binary components are provided under the Apache Software License v
       Amazon Ion Java
       Copyright 2007-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 
+  (ASLv2) AWS Glue Schema Registry Serializer Deserializer
+    The following NOTICE information applies:
+      AWS Glue Schema Registry Serializer Deserializer
+      Copyright 2020 Amazon.com, Inc. or its affiliates.
+
```

Comment:
A line should be added here to reference the NiFi package that contains the copied classes.